dlt/sources/helpers/rest_client/paginators.py

```diff
@@ -846,6 +846,8 @@ def __init__(
         cursor_path: jsonpath.TJsonPath = "cursors.next",
         cursor_param: Optional[str] = None,
         cursor_body_path: Optional[str] = None,
+        stop_after_empty_page: bool = False,
```

Comment:
```suggestion
        *,       
        stop_after_empty_page: bool = False,
```
Let's force keyword-only parameters from here.